client/payment-details/summary/index.tsx

```diff
@@ -113,10 +113,10 @@ const PaymentDetailsSummary = ( {
 		featureFlags: { isAuthAndCaptureEnabled },
 	} = useContext( WCPaySettingsContext );
 
-	// We should only fetch the authorization data if the payment is marked for manual capture
+	// We should only fetch the authorization data if the payment is marked for manual capture and it is not already captured.
 	const shouldFetchAuthorization =
-		charge.amount !== charge.amount_captured &&
-		charge.amount_refunded === 0 &&
+		'captured' in charge && // captured key is only present for manual capture payments. See https://stripe.com/docs/api/charges/object#charge_object-captured
```

Comment:
In my test, I found `captured: false` prop for a failed transaction with manual capture disabled ( automatic capture ) . 

![image](https://user-images.githubusercontent.com/4162931/205919694-05fb6285-a6be-43c9-aea7-f7562af47295.png)

We may want to test if the `captured` key is passed on by Stripe , for a failed transaction, even if it is not a manual capture.

Reply:
Advance apologies, if I have missed anything or not done the test correctly.

Reply:
No need to apologyze, you are 100% true. I misunderstood the Stripe docs here https://stripe.com/docs/api/charges/object#charge_object-captured. So I thought that the captured key would only be present in manual capture, but that is not the case. This means it is not enough to check for the presence of captured, we need to explicitly exclude:
* Failed charges
* Refunds